onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.h

```diff
@@ -236,7 +240,8 @@ namespace Dml
         explicit ExecutionProvider(
             IDMLDevice* dmlDevice,
             ID3D12CommandQueue* commandQueue,
-            bool enableMetacommands = true
+            bool enableMetacommands,
+            bool enableDynamicGraphFusion
```

Comment:
Should we change the order of the parameter to keep the default value of the `enableMetacommands`? Or we can add default value to both of the parameter as `true` and `false` respectively.

Reply:
I'm not a fan of default values here, especially since they were never used. All callers of `ExecutionProvider` were passing their own value for `enableMetacommands` already, and I think we should keep it that way.